scalafix-core/shared/src/main/scala/scalafix/internal/rule/RemoveUnusedImports.scala

```diff
@@ -28,6 +28,6 @@ case class RemoveUnusedImports(index: SemanticdbIndex)
   }
   override def fix(ctx: RuleCtx): Patch =
     ctx.tree.collect {
-      case i: Importee if isUnused(i) => ctx.removeImportee(i)
+      case i: Importee if isUnused(i) => ctx.removeImportee(i).atomic
```

Comment:
Are there any other scalafix rules that could benefit from `.atomic`?

Reply:
Yep, this is what I am doing right now. I'm making sure all the Scalafix rules uses atomic.